src/given.jl

```diff
@@ -0,0 +1,39 @@
+"""
+    `@given d x`
```

Comment:
an alternative could be `@use`? Need to think about names.

Reply:
Also one of my labmates and I came to `@df` independently, I think it's a winner.